changes/544.feature.rst

```diff
@@ -0,0 +1,2 @@
+Modified GUI options in new project wizard to advise on platform compatibility,
+and disallow deployment on incompatible platforms through the create and update commands.
```

Comment:
This should be "539.feature.rst", not 544, so that the issue number tracks the original feature request that is being closed.